Could not find classworlds.jar

When I tried user store first, I encountered the following error:

```
./gradlew assemble

To honour the JVM settings for this build a new JVM will be forked. Please consider using the daemon: https://docs.gradle.org/2.14.1/userguide/gradle_daemon.html.

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'realm-android-user-store'.
> Could not resolve all dependencies for configuration ':classpath'.
   > Could not find classworlds.jar (classworlds:classworlds:1.1-alpha-2).
     Searched in the following locations:
         file:/Users/dalinaum/.m2/repository/classworlds/classworlds/1.1-alpha-2/classworlds-1.1-alpha-2.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
```

I assumed that it is related to a maven setting and I modified the `build.gradle` file as following:

```
    repositories {
//        maven { url "${System.env.HOME}/.m2/repository" }
        jcenter()
    }
```

It built without any errors. How can I build with `maven { url "${System.env.HOME}/.m2/repository" }` successfully?

--- a/build.gradle
+++ b/build.gradle
@@ -6,8 +6,8 @@ project.ext.realmVersion = '2.2.0-SNAPSHOT'
 
 buildscript {
     repositories {
-        maven { url "${System.env.HOME}/.m2/repository" }
         jcenter()
+        maven { url "${System.env.HOME}/.m2/repository" }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.2.0'

